README.md

```diff
@@ -72,3 +72,19 @@ To remove the virtual environment
 make rm_env
 ```
 
+### Troubleshooting
+Ubuntu 20.04 users might experience issues when creating the environment (running `make env`) due to an older version of `setuptools` installed on the virtual environment.
+To work around this issue, update the version of `pipenv` to `2020.11.4` or newer.
+```
+pip install --upgrade --user pipenv
+```
```

Comment:
I tried this workaround , updated my pipenv to  2020.11.15 and the same issue with python-json-logger is present.